app/src/main/java/com/webtutsplus/ecommerceapp/Activity/Home/HomeActivity.java

```diff
@@ -0,0 +1,119 @@
+package com.webtutsplus.ecommerceapp.Activity.Home;
```

Comment:
This should be the main Activity. The existing main activity should be opened by clicking on a button in this homeactivity